Cabal/Cabal.cabal

```diff
@@ -342,6 +342,14 @@ library
   if !impl(ghc >= 7.0)
     default-extensions: CPP
 
+executable cabal-setup
```

Comment:
It's not, because `build-tools` are not considered during dependency resolution; this is bug #220